edc-extensions/dataplane-http-access-control/README.md

```diff
@@ -0,0 +1,49 @@
+# Data Plane: Http Access Control
+
+The goal of this extension is to perform granular access control checks using the Digital Twin Registry (or 
+Digital Twin Registries) configured.
+
+When an HTTP request is targeting a configured Digital Twin Registry, the extension allows the request through.
+
+Otherwise, the extension calls the configured access verification endpoints of the configured Digital Twin 
+Registries and tries to verify that the client has access to the backend asset based on the registries response.
+
+
+## Configuration
+
+```properties
+# Configure the automatically added error endpoint that is used as an alternative target when the client 
+# request cannot be allowed to reach the original backend target
+edc.granular.access.verification.error.endpoint.port=9054
+
+# Configure the URL which is used when the clients are reaching the EDC Data Plane's /public/ endpoint 
+edc.granular.access.verification.edc.data.plane.baseUrl=http://edc-data-plane:9051/public/
+
+# List the names of each DTR instance we intend to configure (comma separated list)
+edc.granular.access.verification.dtr.names=default
+
+# Configure each DTR instance using the following properties.
+# "use the edc.granular.access.verification.dtr.config.<dtr_name>." prefix with each name listed above
+
+# How long should we cache the response we have received from DTR?
+edc.granular.access.verification.dtr.config.default.dtr.decision.cache.duration.minutes=1
+# A RegExp pattern that can match the submodel endpoints in scope for DTR access control
+edc.granular.access.verification.dtr.config.default.aspect.model.url.pattern=http:\/\/aspect-model:8080\/path\/.*
+# The full URL of the access control verification endpoint of the DTR instance
+edc.granular.access.verification.dtr.config.default.dtr.access.verification.endpoint.url=http://dtr:8080/aas-registry-api/v3.0/submodel-descriptor/authorized
+# The OAuth2 configuration for accessing the DTR
+# Token endpoint URL
+edc.granular.access.verification.dtr.config.default.oauth2.token.endpoint.url=http://oauth2-iam:8080/iam/access-management/v1/tenants/00000000-0000-0000-0000-000000000000/openid-connect/token
+# Scope (audience) of the token we want to obtain
+edc.granular.access.verification.dtr.config.default.oauth2.token.scope=aud:dtr
+# OAuth2 client Id
+edc.granular.access.verification.dtr.config.default.oauth2.token.clientId=dtr_client
+# The name (path) of the secret where the OAuth2 client secret is stored in the Vault
+edc.granular.access.verification.dtr.config.default.oauth2.token.clientSecret.path=dtrsecret
+```
+
+## Dependencies
+
+The following header is used from the `HttpDataAddress`:
+
+- `Edc-Bpn`: the BPN of the consumer
```

Comment:
For what reason? Is it always required? And which HttpDataAddress - in Asset or Transfer? Could you possibly fill this area with more details?

Reply:
Thanks, you are right, the documentation part was quite lacking, sorry! I have tried to add some more details in my latest update.